modules/perception/obstacle/camera/transformer/flat_camera_transformer.cc

```diff
@@ -78,5 +97,70 @@ Eigen::Matrix<float, 3, 1> FlatCameraTransformer::MakeUnit(
   return unit_v;
 }
 
+bool FlatCameraTransformer::HaveHighConfidence(
+  std::shared_ptr<VisualObject> obj_ptr) {
+    if (obj_ptr->trunc_width > 0.25f) return false;
+    if (obj_ptr->trunc_height > 0.25f) return false;
+    if (obj_ptr->distance > 50.0) return false;
+    if (obj_ptr->distance < 5.0) return false;
+
+    double azimuth = std::atan2(obj_ptr->center[1], obj_ptr->center[0])
+     * 180.0 / M_PI;
+    if (!(-30.0 < azimuth && azimuth < 30.0)) return false;
```

Comment:
Try convert it into conf or flag if possible.

Reply:
Next fix